### PR TITLE
Updated internationalisation : fr-fr

### PR DESCRIPTION
**Pull request by tostt**
# Comprehensive Summary of your change

Edited file `fr-fr-lang.json`:
- Sorted keys the same way as in `en-us-lang.json`
- Removed keys that are not present in `en-us-lang.json`
- Translated new strings into French

Due to the nature of the changes, no build or test of the application was performed.

Please indicate you've done the following:
- [X] Well Written Title and Summary of the PR
- [x] Label the PR as needed. "release-note/ignore-for-release, release-note/new-feature, release-note/update, release-note/enhancement, release-note/community, release-note/breaking-change, release-note/docs, release-note/infra, release-note/deprecation"
- [x] Accepted the DCO. Commits without the DCO will delay acceptance.
- [x] Made sure tests are passing and test coverage is added if needed.
- [x] Considered the docs impact and opened a new docs issue or PR with docs changes if needed in [website repository](https://github.com/goharbor/website).

### CODE REVIEW

**Review by Vad1mo — APPROVED** [review]

File: src/portal/src/i18n/lang/fr-fr-lang.json [hunk 1]
```diff
@@ -130,6 +134,8 @@
         "CLI_PASSWORD_TIP": "The CLI secret can be used as a password, for Docker or Helm client. With the OIDC auth mode enabled, we strongly recommend using robot accounts, as CLI secrets depend on the validity of the ID token and require the user to regularly log in to the UI to refresh the token.",
```
Comment (thcdrt) [inline src/portal/src/i18n/lang/fr-fr-lang.json:134]: Could you translate this old one too please ? (not required)

File: src/portal/src/i18n/lang/fr-fr-lang.json [hunk 2]
```diff
@@ -327,6 +333,8 @@
         "DELETED_SUCCESS": "Membre supprimé avec succès.",
         "SWITCHED_SUCCESS": "Rôle du membre changé avec succès.",
         "OF": "sur",
+        "SWITCH_TITLE": "Confirmez le changement de membres projet",
+        "SWITCH_SUMMARY": "Voulez-vous changer les membres projet {{param}}?",
```
Comment (thcdrt) [inline src/portal/src/i18n/lang/fr-fr-lang.json:337]: What is containing param ?

File: src/portal/src/i18n/lang/fr-fr-lang.json [hunk 3]
```diff
@@ -287,8 +293,19 @@
...
-        "EMAIL": "Email",
         "ROLE": "Rôle",
+        "SYS_ADMIN": "Administrateur système",
+        "PROJECT_ADMIN": "Administrateur de projet",
+        "PROJECT_MAINTAINER": "Mainteneur du projet",
+        "DEVELOPER": "Développeur",
+        "GUEST": "Invité",
+        "LIMITED_GUEST": "Invité avec droits limités",
+        "DELETE": "Supprimer",
+        "ITEMS": "entrées",
```
Comment (thcdrt) [inline src/portal/src/i18n/lang/fr-fr-lang.json:304]: Sometimes item is translated by "élément" and sometimes by "entrée", we should be consistent I think

File: src/portal/src/i18n/lang/fr-fr-lang.json [hunk 4]
```diff
@@ -877,6 +893,8 @@
             "PRO_CREATION_RESTRICTION": "L'indicateur pour définir quels utilisateurs ont le droit de créer des projets. Par défaut, tout le monde peut créer un projet. Définissez sur 'Administrateur Seulement' pour que seul un administrateur puisse créer un projet.",
             "ROOT_CERT_DOWNLOAD": "Télécharger le certificat racine du dépôt.",
             "SCANNING_POLICY": "Définissez la politique d'analyse des images en fonction des différentes exigences. 'Aucune' : pas de politique active; 'Tous les jours à' : déclenchement de l'analyse à l'heure spécifiée tous les jours.",
+            "VERIFY_CERT": "Verifier le certificat par le serveur LDAP",
```
Comment (thcdrt) [inline src/portal/src/i18n/lang/fr-fr-lang.json:896]:
```suggestion
            "VERIFY_CERT": "Verifier le certificat du serveur LDAP",
```

File: src/portal/src/i18n/lang/fr-fr-lang.json [hunk 5]
```diff
@@ -1187,6 +1211,8 @@
         "DELETE_REPLICATION": "Suppression de réplication",
         "DELETE_MEMBER": "Suppression de membre",
         "DELETE_GROUP": "Suppression de membre d'un groupe",
+        "DELETE_CHART_VERSION": "Supprimer version de chart",
```
Comment (thcdrt) [inline src/portal/src/i18n/lang/fr-fr-lang.json:1214]:
```suggestion
        "DELETE_CHART_VERSION": "Supprimer la version de chart",
```
?

File: src/portal/src/i18n/lang/fr-fr-lang.json (hunk 2 above)
Comment (thcdrt) [reply src/portal/src/i18n/lang/fr-fr-lang.json:337]: I found the formulation weird

File: src/portal/src/i18n/lang/fr-fr-lang.json (hunk 1 above)
Comment (tostt) [reply src/portal/src/i18n/lang/fr-fr-lang.json:134]: I missed that one. I will add it.

File: src/portal/src/i18n/lang/fr-fr-lang.json (hunk 3 above)
Comment (tostt) [reply src/portal/src/i18n/lang/fr-fr-lang.json:304]: Yes, élément seems more appropriate. I will correct it.

File: src/portal/src/i18n/lang/fr-fr-lang.json (hunk 2 above)
Comment (tostt) [reply src/portal/src/i18n/lang/fr-fr-lang.json:337]: This text doesn't seem to be used at all by the application. I can't find it referenced anywhere. So for now, I will leave it as it is.

File: src/portal/src/i18n/lang/fr-fr-lang.json (hunk 3 above)
Comment (tostt) [reply src/portal/src/i18n/lang/fr-fr-lang.json:304]: I will change FILTER_ITEMS so all ITEMs are "entrée".

**Review by thcdrt — APPROVED** [review]
Thank you, LGTM

**Comment by OrlinVasilev** [comment]
@tostt congrats on merging that first PR here and now :)

**Pull request merged**